### DISCUSSION TITLE
There are suggestions for compatibility between the godot and steam platforms

### DISCUSSION THREAD

**Discussion by yuho170** [godotengine/godot-proposals#7536] [Editor]
Thank you for checking out and responding to this post.

One problem that bothers me is using Godot on Steam. When you run the game, every time it pops up in the bottom right corner, sometimes it even distorts. It's not a big deal, but it's a little annoying. I don't know how to set up steam. If it can be turned off, please let me know. If this is something that can't be solved by individuals, can godot optimize it?

Thanks again.

**Comment by sonic2kk** [comment]
> When you run the game, every time it pops up in the bottom right corner

Are you referring to how Steam alerts people on your friends list, stating "[Name] is now in-app Godot Engine"? 

If so, while this cannot be turned off per-se, if you set your Steam visibility to "Invisible", others will not get notifications when you launch Godot or any other application from Steam. It will still be visible on your Steam profile that you have launched the application, and if you are using Godot installed from Steam and not as a non-Steam game, it will still show up on your profile that you have launched the game previously, as well as what date you launched it on, and the total hours you have spent using Godot on Steam.

It is also possible for others on your friends list to turn off those notifications, globally or per-friend.

If you're referring to the pop-up for the Steam Overlay in the bottom right, this can be configured in the application properties from Steam (right click the Godot entry in your Steam library -> Properties...). There should be an option under "General" to disable the Steam Overlay. Though ou should note that this could cause some issues with things like controller detection or process management.

If you want to avoid either of these altogether, Godot is available from other sources, such as a direct executable for all major desktop operating systems (Desktop Linux distributions with X11 capabilities, modern versions of macOS, and Windows 10+)

**Reply by yuho170** [reply]
Thank you, I successfully closed it, I mean it will always remind us to CTRL+SHIFT to open the community. This is annoying to listen to every time you test a game.
Thank you again!

If you have the same question:

right click the Godot entry in your Steam library -> Properties -> Close Open the steam interface in the game

**Reply by Calinou** [reply]
As an aside, it's possible for certain games to force Steam overlay notifications to appear in a specific corner, but I don't know how it's done or whether individual developers can even choose this.

For example, CS:GO always displays Steam notifications in the top-left corner (instead of the default bottom-right corner) so that the ammo counter on the HUD is never covered by Steam notifications.

**Reply by sonic2kk** [reply]
> but I don't know how it's done or whether individual developers can even choose this.

I think there's a global Steam setting to choose which corner the notification appears in by default (including Steam Screenshot notifications, achievement progress/unlocks, messages, etc) but I think games can override this using some Steam API, for similar reasons to what you outlined. However I don't believe games can disable this wholly.
